### PR TITLE
Scrape rpilocator.com sample

### PR DESCRIPTION
**Pull request by S4adam**
Scrape https://rpilocator.com/ and send url of the new stock to the Stream.

### CODE REVIEW

File: python/rpilocator-scraper/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# Install dependencies
+npm run build
+
+# Deploy sample to STH
+si seq deploy dist
+
+```
+
+If any new stock comes up, it will be written into the stream.
```
Comment (alicja-gruzdz) [inline python/rpilocator-scraper/README.md:43]: I'd add here how to obtain the data stream. If we wish to write to a topic, then how to obtain topic data by CLI:
si topic get <topicName>

File: python/rpilocator-scraper/main.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import asyncio
+import feedparser
+from scramjet.streams import Stream
+
+
+# it will send data to the 'telegram-inbound' Topic
+provides = {
+    'provides': 'telegram-inbound',
```
Comment (alicja-gruzdz) [inline python/rpilocator-scraper/main.py:8]: leftover topic name :)

File: python/rpilocator-scraper/main.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+async def scrape_rp(stream):
+    first_check = feedparser.parse(URL).entries
+    stock_ids=[entry.id for entry in first_check]
+
+    await asyncio.sleep(DELAY)
+
+    while True:
+        new_data = feedparser.parse(URL)
+        for entry in new_data.entries:
+            if entry.guid not in stock_ids:
```
Comment (alicja-gruzdz) [inline python/rpilocator-scraper/main.py:32]: when not in stock or in stock?

**Review by alicja-gruzdz — COMMENTED** [review]
I've tried to run it with a query:
```
# specify models:
MODELS = "CM4"

# and regions:
REGIONS = "UK,US,CA,DE,PL"
```

but it found no entries :( 
The entries were loaded at init into `stock_ids`, but not written to the stream.

File: python/rpilocator-scraper/main.py (hunk 3 above)
Comment (alicja-gruzdz) [reply python/rpilocator-scraper/main.py:32]: ok, I think I know what was meant. Not in stock, but shouldn't we write to the stream the entries which were found initially?

Comment (S4adam) [reply python/rpilocator-scraper/main.py:36]: Initially, I didn't want to write `first_data ` entries to the stream but I believe you are right. Changed it just now.

Comment (alicja-gruzdz) [reply python/rpilocator-scraper/main.py:36]: thanks :)

**Review by alicja-gruzdz — APPROVED** [review]

**Pull request merged**